README.md

```diff
@@ -29,6 +29,20 @@ export { reportWebVitals } from 'next-axiom';
 
 ## Sending Logs
 
+:warning: If you log from a function, please call `await log.flush()` at the end
```

Comment:
Can we start this section with a few examples without flush and then have a `Flushing logs` section that explains when and how to flush logs?

Reply:
done, please check. Its probably not the best writing though.